src/Microsoft.DotNet.Wpf/src/PresentationCore/System/Windows/Navigation/BaseUriHelper.cs

```diff
@@ -13,62 +14,47 @@
-
 namespace System.Windows.Navigation
 {
     /// <summary>
     /// BaseUriHelper class provides BaseUri related property, methods.
     /// </summary>
     public static class BaseUriHelper
     {
-        private const string SOOBASE = "SiteOfOrigin://";
-        private static readonly Uri _siteOfOriginBaseUri = PackUriHelper.Create(new Uri(SOOBASE));
-        private const string APPBASE = "application://";
-        private static readonly Uri _packAppBaseUri = PackUriHelper.Create(new Uri(APPBASE));
+        private static readonly Uri s_siteOfOriginBaseUri = PackUriHelper.Create(new Uri("SiteOfOrigin://"));
+        private static readonly Uri s_packAppBaseUri = PackUriHelper.Create(new Uri("application://"));
 
-        private static Uri _baseUri;
+        private const string COMPONENT = ";component";
+        private const string VERSION = "v";
+        private const char COMPONENT_DELIMITER = ';';
+
+        private static Assembly s_resourceAssembly;
 
-        // Cached result of calling
-        // PackUriHelper.GetPackageUri(BaseUriHelper.PackAppBaseUri).GetComponents(
-        // UriComponents.AbsoluteUri,
-        // UriFormat.UriEscaped);
-        private const string _packageApplicationBaseUriEscaped = "application:///";
-        private const string _packageSiteOfOriginBaseUriEscaped = "siteoforigin:///";
+        // Cached result of calling the following:
+        // PackUriHelper.GetPackageUri(BaseUriHelper.PackAppBaseUri).GetComponents(UriComponents.AbsoluteUri, UriFormat.UriEscaped);
+        private const string PackageApplicationBaseUriEscaped = "application:///";
+        private const string PackageSiteOfOriginBaseUriEscaped = "siteoforigin:///";
 
         static BaseUriHelper()
         {
-            _baseUri = _packAppBaseUri;
             // Add an instance of the ResourceContainer to PreloadedPackages so that PackWebRequestFactory can find it
             // and mark it as thread-safe so PackWebResponse won't protect returned streams with a synchronizing wrapper
             PreloadedPackages.AddPackage(PackUriHelper.GetPackageUri(SiteOfOriginBaseUri), new SiteOfOriginContainer(), true);
         }
 
-        #region public property and method
-
         /// <summary>
         ///     The DependencyProperty for BaseUri of current Element.
         ///
         ///     Flags: None
         ///     Default Value: null.
         /// </summary>
-        public static readonly DependencyProperty BaseUriProperty =
-                    DependencyProperty.RegisterAttached(
-                                "BaseUri",
-                                typeof(Uri),
-                                typeof(BaseUriHelper),
-                                new PropertyMetadata((object)null));
-
+        public static readonly DependencyProperty BaseUriProperty = DependencyProperty.RegisterAttached("BaseUri", typeof(Uri), typeof(BaseUriHelper), new PropertyMetadata(null));
```

Comment:
Removing the cast to object means that this code will use [this constructor](https://learn.microsoft.com/en-us/dotnet/api/system.windows.propertymetadata.-ctor?view=windowsdesktop-9.0#system-windows-propertymetadata-ctor(system-windows-propertychangedcallback)).

I'm not sure if it changes the behavior but might as well keep the same behavior and you can juste use a named parameter (Which IMO is also cleaner than just having null):
```suggestion
        public static readonly DependencyProperty BaseUriProperty = DependencyProperty.RegisterAttached("BaseUri", typeof(Uri), typeof(BaseUriHelper), new PropertyMetadata(defaultValue: null));
```

Reply:
Also, what's wrong with the original indentation ? Most dependency/attached property registration in this repo use the same indentation style, I don't see why we should change it. Personally I prefer the previous style and other repos like WinForms have a recommended max line length ([WinForms uses recommended < 120 and max < 150](https://github.com/dotnet/winforms/blob/main/docs/coding-style.md#line-breaks))

Reply:
It won't change behaviour but I like the named argument there, let's use that. Thanks.

As for indentation; since the class is 90% changes, it was auto format. Reverted it as the line is indeed too long but I wouldn't go as far as saying "use the same indentation style", I can select a random file e.g. `DataGrid` files and I'll see 6 different ways to indent a dependency property. 😀